.github/workflows/main.yml

```diff
@@ -0,0 +1,57 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go: [ '1.14', '1.*' ]
+    name: Tests
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go }}
+      - name: Get dependencies
+        run: go get -v -t -d ./...
```

Comment:
We could cache dependencies with [actions/cache](https://github.com/actions/cache). However, it only saves us a few seconds so I'm not sure if it's worth it. What do you think?

Reply:
Yeah, let's discuss this in our next contributor's meetup.

Reply:
the download is quite fast, so let's skip this for now